crates/utils/re_mcap/src/parsers/ros2msg/rcl_interfaces/log.rs

```diff
@@ -0,0 +1,144 @@
+        match level {
+            LogLevel::Info => Color::from(Rgba32::from_rgb(0, 128, 255)), // Blue
+            LogLevel::Warn => Color::from(Rgba32::from_rgb(255, 165, 0)), // Orange
+            LogLevel::Error => Color::from(Rgba32::from_rgb(255, 0, 0)),  // Red
+            LogLevel::Fatal => Color::from(Rgba32::from_rgb(139, 0, 0)),  // Dark Red
+            LogLevel::Unknown | LogLevel::Debug => {
+                Color::from(Rgba32::from_rgb(128, 128, 128)) // Gray
+            }
+        }
+    }
+}
+
+impl MessageParser for LogMessageParser {
+    fn append(&mut self, ctx: &mut ParserContext, msg: &mcap::Message<'_>) -> anyhow::Result<()> {
+        re_tracing::profile_function!();
+        let rcl_interfaces::Log {
+            stamp,
+            level,
+            name,
+            msg: log_msg,
+            file,
+            function,
+            line,
+        } = cdr::try_decode_message::<rcl_interfaces::Log>(&msg.data)
+            .context("Failed to decode `rcl_interfaces::Log` message from CDR data")?;
+
+        // add the sensor timestamp to the context, `log_time` and `publish_time` are added automatically
+        ctx.add_time_cell(
+            "timestamp",
+            TimeCell::from_timestamp_nanos_since_epoch(stamp.as_nanos()),
+        );
+
+        self.text_entries.push(format!("[{name}] {log_msg}"));
+        self.levels.push(level.to_string());
+        self.colors.push(Self::ros2_level_to_color(level));
+
+        self.file.values().append_value(file);
+        self.file.append(true);
+
+        self.function.values().append_value(function);
+        self.function.append(true);
+
+        self.line.values().append_slice(&[line]);
+        self.line.append(true);
+
+        Ok(())
+    }
+
+    fn finalize(self: Box<Self>, ctx: ParserContext) -> anyhow::Result<Vec<Chunk>> {
+        re_tracing::profile_function!();
+        let Self {
+            text_entries,
+            levels,
+            colors,
+            mut file,
+            mut function,
+            mut line,
+        } = *self;
+
+        let entity_path = ctx.entity_path().clone();
+        let timelines = ctx.build_timelines();
+
+        let text_components: Vec<Text> = text_entries.into_iter().map(Text::from).collect();
+        let level_components: Vec<TextLogLevel> =
+            levels.into_iter().map(TextLogLevel::from).collect();
+
+        let text_log = TextLog::update_fields()
+            .with_many_text(text_components)
+            .with_many_level(level_components)
+            .with_many_color(colors);
+
+        let mut chunk_components: Vec<SerializedComponentColumn> =
+            text_log.columns_of_unit_batches()?.collect();
+
+        chunk_components.extend([
+            Self::create_metadata_column("file", file.finish()),
+            Self::create_metadata_column("function", function.finish()),
+            Self::create_metadata_column("line", line.finish()),
+        ]);
```

Comment:
I think we should add these to the official `TextLog` archetype - they make a lot of sense to standardize